openmetadata-docs/content/connectors/dashboard/domo-dashboard/airflow.md

```diff
@@ -20,6 +20,13 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+For metadata ingestion, kindly make sure add alteast `dashboard` scopes to the clientId provided.
+Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5)
```

Comment:
```suggestion
Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5).
```

Reply:
nit: all those links do not finish with a dot at the end. We can be more consistent with the format of the text.